translations/translations.json

```diff
@@ -2170,6 +2171,7 @@
       "address": "Address",
       "date": "Date",
       "events": "Events",
+      "exportAsICal": "export as iCal",
```

Comment:
I just realized that the design suggests different label
https://www.figma.com/file/cA4F2MwHs2LNGviOWUjfE9/Integreat-Design-Tickets-Jira?node-id=501%3A724&t=LhRdZbuawcdhRVsl-0

Reply:
Yes it's different because the first idea was to trigger an event import of the calendar directly which isn't possible from web. Therefore the more accurate label text is "export as iCal"